src/main/java/com/rednavis/vaadin/showcase/backend/db/DbConstants.java

```diff
@@ -0,0 +1,11 @@
+package com.rednavis.vaadin.showcase.backend.db;
+
+public class DbConstants {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'DbConstants' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4827083)